ArkKit/Ark.swift

```diff
@@ -57,13 +58,19 @@ class Ark {
             arkState.setup(stateSetupFunction)
         }
     }
+    
+    private func setupDefaultEntities() {
+        arkState.arkECS.createEntity(with: [StopWatchComponent(name: "_ArkWorldTime")])
```

Comment:
probably better to reference the name via a constant variable instead of a magic literal like this

possible to make the `ARK_WORLD_TIME` constant variable below a static variable of the `ArkPhysicsSystem`, then reference it here as `ArkPhysicsSystem.ARK_WORLD_TIME` into the name component